build.sh

```diff
@@ -281,7 +274,7 @@ buildpy () {
   then
     rm -rf "$server_repo"
   fi
-  git clone $REPO_ORGANIZATION/server.git \
+  git clone https://github.com/triton-inference-server/server.git \
```

Comment:
Why are we hardcoding this here now?

Reply:
I changed it back from before #404. I was not sure where build.sh is used in the pipeline so I let it be hardcoded, like before